connect-inject/container_init.go

```diff
@@ -323,6 +323,19 @@ func (h *Handler) containerInit(pod *corev1.Pod, k8sNamespace string) (corev1.Co
 // and the connect-proxy service should come after the "main" service
 // because its alias health check depends on the main service to exist.
 const initContainerCommandTpl = `
+{{- if .AuthMethod }}
+consul-k8s connect-init -method="{{ .AuthMethod }}" \
+  {{- if.ConsulNamespace }}
+  {{- if .NamespaceMirroringEnabled }}
+  {{- /* If namespace mirroring is enabled, the auth method is
+         defined in the default namespace */}}
+  -namespace="default" \
+  {{- else }}
+  -namespace="{{ .ConsulNamespace }}" \
+  {{- end }}
+  {{- end }}
+  -meta="pod=${POD_NAMESPACE}/${POD_NAME}"
+{{- end }}
 {{- if .ConsulCACert}}
 export CONSUL_HTTP_ADDR="https://${HOST_IP}:8501"
```

Comment:
exports need to be first

Reply:
great catch thx! got mixed up during a refactor from the PR